src/components/Filezone.tsx

```diff
@@ -1,12 +1,12 @@
 export default function Example() {
```

Comment:
gotchya. since this is how it looks on develop, and this branch only addresses styling, i'm just going to merge and i can add a quick branch to address this syntax for better organization.